nrt/nrt.py

```diff
@@ -582,6 +608,12 @@ def main():
                                 keep_job = False
                                 break
 
+                    if keep_job and (len(directory_failed) > 0):
+                        if job_file in directory_failed:
+                            keep_job = True
```

Comment:
The previous value of keep_job should be kept (if for example we only want to run a sublist of the failed tests)

Reply:
taken into account